.github/workflows/integration-tests-benchmarks.yml

```diff
@@ -8,7 +8,7 @@ on:
       - '**/sentry-android/**'
       - '**/sentry-android-core/**'
       - '**/sentry-android-ndk/**'
-      - '**/sentry-android-integration-tests/sentry-uitest-android-benchmark/**'
+      - '**/sentry-android-integration-tests/**'
```

Comment:
I'd avoid using `'**/sentry-android-integration-tests/**'`, since we don't want to run benchmarks if we only add a ui tests (the module `sentry-android-integration-tests/sentry-uitest-android`)